decidim-forms/app/views/decidim/forms/questionnaires/answers/_matrix_single.html.erb

```diff
@@ -1,44 +1,46 @@
-<table class="questionnaire-question-matrix table">
-  <thead>
-    <tr>
-      <td></td>
-      <% answer.question.answer_options.each_with_index do |answer_option, idx| %>
-        <td><%= translated_attribute(answer_option.body) %></td>
-      <% end %>
-    </tr>
-  </thead>
-  <tbody>
-    <% answer.question.matrix_rows.by_position.each_with_index do |row, row_idx| %>
-      <tr class="radio-button-collection">
-        <td><%= translated_attribute row.body %></td>
+<div class="answer-questionnaire__single-matrix">
```

Comment:
When I add more question columns the grid questions (both single and multiple) don't have proper vertical scrolling. As a reference point, here is how the big G does it with their forms:

https://user-images.githubusercontent.com/864340/183113936-ebaa507d-97b1-4769-aad8-098c5639ca67.mp4